packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -45,8 +45,9 @@ const getGatsbyImageCdnFields = async ({
   }
 
   const mimeType = node.attributes.filemime
+  const { filename } = node.attributes
 
-  if (!mimeType) {
+  if (!mimeType || !filename) {
```

Comment:
I wonder how often filename is empty and if so if we should do something like
```
filename = filename ?? `file.${mimeType/split('/').pop()}`
```

Reply:
wont we get clashing file paths when 2 different images have the same args? Seems pretty likely since you have the same query for many images

Reply:
I think it's safer to ignore those images and warn - since it'd be corrupt data in the db

Reply:
Ok, cool 👍 the name will always be there unless something in Drupal is broken because the file field adds the filename on upload.